doc/content/integrations/node-red/receive/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Receive Events and Messages"
+description: ""
+weight: 2
+---
+
+This section follows the process of setting up a flow which subscribes and listens to the events and messages that are being published by the MQTT Server. 
+
+## Configure MQTT In Node
+
+Find the **mqtt in** node in the library section and place it on the flow dashboard. This node connects to the MQTT broker and subscribes to messages from the specified topic. Double-click on the node to configure its properties.
+
+{{< figure src="mqtt_in_node_properties.png" alt="mqtt in node properties" >}}
```

Comment:
This screenshot already contains a configured `localhost:8883` server. Please remove, as it will be added in instructions below

Reply:
Also please put the screenshots under the instructions, instead of the screenshots above

Reply:
Second that. It's not clear where to look on the printscreens. It takes too much time to look for which buttons to press and where the add the content.